### PR TITLE
refactor: nullable and default in schema generate

### PR DESCRIPTION
**Pull request by aiwantaozi**
**Problem:**
1. missing default value in some case while generate schema
2. nullable cause UI validate error

**Solution:**
1. Fix the default value generated
2. remove nullable

**Related Issue:**
#1356 
#1688

### CODE REVIEW

File: pkg/templates/loader/testdata/with_output/expected.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ openAPISchema:
             x-walrus-ui:
               colSpan: 12
               order: 1
+              widget: YamlEditor
```
Comment (gitlawr) [inline pkg/templates/loader/testdata/with_output/expected.yaml:16]: What if we don't set widget in schema? Will UI fall back to use the yaml editor?

File: pkg/templates/translator/translator.go [hunk 2]
```diff
@@ -18,13 +18,13 @@ type Translator interface {
...
-	Name        string
-	Default     any
-	Description string
-	Sensitive   bool
-	Order       int
-	TypeExpress any
-	Nullable    bool
+	Name          string
+	DefaultValue  any
+	DefaultObject any
```
Comment (gitlawr) [inline pkg/templates/translator/translator.go:23]: Why do we need both fields?

Comment (aiwantaozi) [reply pkg/templates/translator/translator.go:23]: DefaultObject is used for passing the default for nested object.

**Review by gitlawr — APPROVED** [review]
LGTM

**Review by shanewxy — APPROVED** [review]
LGTM

**Pull request merged**